apps/fxc-front/src/app/logic/track.ts

```diff
@@ -1,9 +1,21 @@
-import { extractGroupId } from '@flyxc/common';
+import {extractGroupId, RuntimeTrack} from '@flyxc/common';
 
 import { unwrapResult } from '@reduxjs/toolkit';
 
-import { store } from '../redux/store';
-import { fetchTrack } from '../redux/track-slice';
+import {AppDispatch, store} from '../redux/store';
+import { currentLeague } from '../redux/selectors';
+import {fetchTrack, RuntimeTrackId, setScore as setTrackStore} from '../redux/track-slice';
+// @ts-ignore
+import ScoreWorker from '../workers/score-track?worker';
+import {Request as ScoreRequest, Response as ScoreResponse} from '../workers/score-track'
+import {
+  setIsFreeDrawing as setPlannerIsFreeDrawing,
+  setEnabled as setPlannerEnabled,
+  setRoute as setPlannerRoute,
+  setScore as setPlannerScore
+} from '../redux/planner-slice';
+import { Point, ScoreInfo } from 'igc-xc-score';
```

Comment:
Maybe keep `igc-xc-score` specific type inside the worker and output only flyxc types?
(That should work for a `Point` but `ScoreInfo` might be too specific and we could keep it for now)

Reply:
The igc-xc-score types are now hidden from the other parts of the code (They only appear in apps/fxc-front/src/app/workers/score-track.ts and apps/fxc-front/src/app/logic/score/improvedScorer.ts)